### PR TITLE
Add React Native tests

### PR DESCRIPTION
**Pull request by acostalima**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,39 +1,71 @@
...
+            - uses: codecov/codecov-action@v1
 
+    webkit_macos:
+        name: "Webkit Mac"
+        runs-on: macos-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: microsoft/playwright-github-action@v1
+            - uses: bahmutov/npm-install@v1
+            - run: npx playwright-test --runner tape --browser webkit test.js
```
Comment (hugomrdias) [inline .github/workflows/main.yml:29]:
```suggestion
            - run: npx playwright-test --runner tape --browser webkit
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,39 +1,71 @@
...
-    - uses: bahmutov/npm-install@v1
-    - run: npx playwright-test --runner tape --browser webkit
+    firefox_linux:
+        name: "Firefox Linux"
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: microsoft/playwright-github-action@v1
+            - uses: bahmutov/npm-install@v1
+            - run: npx playwright-test --runner tape --browser firefox test.js
```
Comment (hugomrdias) [inline .github/workflows/main.yml:38]:
```suggestion
            - run: npx playwright-test --runner tape --browser firefox
```

File: rn-test.config.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+module.exports = {
+  require: require.resolve('./rn-test.require.js'),
+  runner: 'zora',
+  modules: [
+    'react-native-url-polyfill'
+  ]
```
Comment (hugomrdias) [inline rn-test.config.js:8]: can you explain this ?

**Review by hugomrdias — CHANGES_REQUESTED** [review]

File: rn-test.config.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+'use strict'
```
Comment (hugomrdias) [inline rn-test.config.js:1]: can you move this to a `config` folder ?

File: rn-test.require.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+'use strict'
```
Comment (hugomrdias) [inline rn-test.require.js:1]: can you move this to a `config` folder ?

**Review by hugomrdias — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (acostalima) [reply .github/workflows/main.yml:29]: Won't `pw-test` pick up `react-native.test.js` this way?

File: .github/workflows/main.yml (hunk 2 above)
Comment (acostalima) [reply .github/workflows/main.yml:38]: Won't `pw-test` pick up `react-native.test.js` this way?

File: rn-test.config.js (hunk 3 above)
Comment (acostalima) [reply rn-test.config.js:8]: React Native's URL implementation provides little functionality. Without the polyfill, the tests won't pass.

File: .github/workflows/main.yml (hunk 1 above)
Comment (hugomrdias) [reply .github/workflows/main.yml:29]: ah correct

File: rn-test.config.js (hunk 3 above)
Comment (hugomrdias) [reply rn-test.config.js:8]: why isnt this in the other file with the other polyfills ?

Comment (acostalima) [reply rn-test.config.js:8]: What file? The polyfill is injected in the require script.

File: rn-test.config.js (hunk 4 above)
Comment (acostalima) [reply rn-test.config.js:1]: `lilconfig` won't pick this up that way. :(

**Pull request merged**